cts.schema.json

```diff
@@ -63,6 +72,10 @@
       "type": "array",
       "description": "The expected result of applying the selector to the document, contains all the matched values"
     },
+    "test_case_results": {
+      "type": "array",
```

Comment:
@gregsdennis Is it possible to specify that the type of "test_case_results" is an array of "test_case_result"?

Reply:
Yes.  Add `"items": {"$ref": "#/$defs/test_case_result"}`.

Reply:
@gregsdennis Excellent! Done. Also, I made the size of the array at least two. (If there's only one possible result, the testcase should be using "result" instead.)